code_samples/bigdecimal_sample.js

```diff
@@ -43,20 +43,19 @@ function portableFactory(classId) {
 }
 
 (async () => {
-    try {
-        const cfg = {
+    const cfg = {
             serialization: {
                 portableFactories: {
                     1: portableFactory
                 }
             }
-        };
+    };
 
-        const client = await Client.newHazelcastClient(cfg);
-        const mapName = 'decimalMap';
-        const map = await client.getMap(mapName);
-        // To be able to use our map in SQL we need to create mapping for it.
-        const createMappingQuery = `
+    const client = await Client.newHazelcastClient(cfg);
+    const mapName = 'decimalMap';
```

Comment:
Let's either use a random map name or use CREATE OR REPLACE MAPPING or something similar to make the consecutive runs easier

Reply:
I think that is done in https://github.com/hazelcast/hazelcast-nodejs-client/pull/1162

Reply:
Ah sorry, I meant to say please see this comment: https://github.com/hazelcast/hazelcast-nodejs-client/pull/1162#discussion_r791716878

Reply:
OK, I can accept constant map names, but then why don't we use CREATE OR REPLACE MAPPING so that the user can run the code multiple times, without an error?

Reply:
Hmm I think I should align map names in all code samples. I will try to make all of them constant and use create or replace mapping. It is a little bit annoying to ensure they are unique, but being constant will be good for users to inspect the map afterwards like Yuce said. Maybe I should use the file name as part of the map name.

Reply:
Ah sorry, we are starting a new member for each test already. 🤦🏻‍♂️  Going to make all of them constant and add CREATE OR REPLACE MAPPING. It is enough to avoid clash in a single file. https://github.com/hazelcast/hazelcast-nodejs-client/blob/0af1d2037ca253cdb82584de1be22de8927b960b/scripts/code-sample-checker.js#L53